templates/solutions/infrastructure/_form_infrastructure.html

```diff
@@ -0,0 +1,89 @@
+<div class="p-modal" style="display: none;" id="infrastructure-modal">
+  <div class="p-modal__dialog is-paper" role="dialog">
+    <div class="js-pagination js-pagination--1">
+      <div class="js-formfield">
+      <div class="row--50-50 p-strip is-shallow">
+        <div class="col">
+          <h2 class="p-heading--1">Contact Canonical</h2>
+          <p>Fill in this form and a member of our team will be in touch shortly.</p>
+        </div>
+        <div class="col">
+          <button class="p-modal__close p-modal-close-button " aria-controls="infrastructure-modal" aria-label="Close active modal" style="margin-top: 0;">Close</button>
```

Comment:
Again, I'd suggest using utility instead of local styles.

Reply:
This one isn't actually in scope of the ticket (left a comment above), I just moved the file for consistency. I'll update though